packages/widgetbook_cli/bin/commands/publish.dart

```diff
@@ -210,34 +213,16 @@ class PublishCommand extends WidgetbookCommand {
   }
 
   @visibleForTesting
-  Future<String?> gitProviderSha({
-    required GitDir gitDir,
-  }) async {
-    final commits = await gitDir.commits();
```

Comment:
The `commits()` method is no longer used in anywhere of our codebase, and can be safely removed with it's internal implementation methods and tests. This will be done in a separate PR to allow reverting in case it's needed.

Reply:
Feel free to remove the method.